docs/ed-fi-badges/2-badging-requirements-for-technology-partners.md

```diff
@@ -0,0 +1,35 @@
+---
+author: Ecleamus Ricks
+tags: [Ed-Fi, Badging, Technology Providers]
+description: Requirements for technology providers to be eligible for Ed-Fi badges.
+created: 2024-08-07
+---
+
+# Badging Requirements for Technology Providers
+:::tip
+
+🚧 This site is under construction. In the meantime, please see
+[Ed-Fi API Guidelines](https://edfi.atlassian.net/wiki/spaces/EFAPIGUIDE/overview)
+for more information.
+
+:::
+
+To be eligible for any Ed-Fi badge, the following requirements must be met. Note that requirements for specific badges are described in Available Ed-Fi Badges.
+
+## Partner Program
+Vendors must submit a request to join the Partner Program by providing the information and commitments requested in this online form. Direct any questions to Sean Casey.
```

Comment:
Please check that you have the recommended VS Code extensions, and then try reformatting this page using markdownlint.